src/exports.jl

```diff
@@ -1068,6 +1068,7 @@ export minimal_block_reps
 export minimal_faces
 export minimal_generating_set
 export minimal_size_generating_set, has_minimal_size_generating_set, set_minimal_size_generating_set
+export minimal_supercone
```

Comment:
Do we need to export this?

Reply:
@simonbrandhorst recommended to me to export it since the code is documented, general and reasonably fast